recipe/patches/0010-Empty-install-requirements-installed-by-conda-build.patch

```diff
@@ -1,14 +1,14 @@
-From 3c5886d8d47152735536fc7809908bbbad8041dd Mon Sep 17 00:00:00 2001
-From: Vasily Litvinov <vasilij.n.litvinov@intel.com>
-Date: Fri, 4 Dec 2020 12:49:34 +0300
+From 353efadf23e790e831f87021ae5d4517331e9a2d Mon Sep 17 00:00:00 2001
+From: Kai Fricke <kai@anyscale.com>
+Date: Fri, 6 Aug 2021 11:36:40 +0100
```

Comment:
Sidenote: we try to keep the attribution to the original authors when rebasing patches; less for the glory and more for knowing how to ask about it ;-)

Reply:
Ah sorry! I just `git am`'d everything and created new patches with `git format-patch` which seems to have lost the original authors. I'll try to sort this out

Reply:
Ah so this was only in this patch, and because I actually didn't `git am` it after all (because of conflicts) but manually did the required changes again and committed with the same commit message. What is the best way to go about this next time?

Reply:
> What is the best way to go about this next time?

For me personally, I keep a branch in a local copy of the upstream repo that I rebase onto new versions, and if a conflict occurs, I try to use `git commit -m "<original_message>" --author="<original_author>" --date="<original_date>"` after resolving possible conflicts (if none exist, it'll be fine already).